### ISSUE TITLE
Doesn't seem to work with .NET Core 3.1

### ISSUE THREAD

**Issue by pkunze** [ritterim/stuntman#186]
I tried to include this in a project based on dotnet core 3.1 
It doesn't throw any errors, but I am not able to select stuntman users either. 
It remains in "Anonymous".

I have attached a sample project which reproduces the issue. 

Please let me know if I can do anything more to resolve this. 
If someone pointed me in the right direction I'd gladly try to solve this myself and submit a PR.

[StuntmanTest.zip](https://github.com/ritterim/stuntman/files/4202957/StuntmanTest.zip)

**Comment by kendaleiv** [comment]
.NET Core 3.1 currently isn't supported (at least, not in any intentional fashion). I think we'd want to add `netcoreapp3.0` to `TargetFrameworks` and make any changes necessary for that version (assuming this will enable both .NET Core 3.0 and .NET Core 3.1 to work).

**Comment by pkunze** [comment]
Alright, i try to make this work. I forked the project and added the target framework as you suggested. 
I narrowed the problem down to `context.SignInAsync` not setting the User/Identity when there is another AuthMiddleware (Cookies) registered. (Sorry if this is phrased in a confusing way, I am not a native speaker and have no clue of Authentication whatsoever ;))

So basically, when using it in a blank .NET Core 3 Project without Authentication it works.
If I use it in a Project I created using the "Individual User Accounts) Auth-Option it doesn't.

I have no clue why and so far, I didn't manage to find the solution using my favorite Search Engines.
So my question is: Would it be of any help if i create a PR with my WIP or does anybody has an idea whats going on/wrong?
